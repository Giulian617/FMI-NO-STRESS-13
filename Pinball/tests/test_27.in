100 59491590604671
72 95 1
2 95 6
59 72 3
52 72 6
97 52 8
55 97 6
94 97 7
88 2 9
76 97 2
44 59 10
1 55 1
92 88 9
20 97 8
87 95 7
38 52 9
12 2 1
13 59 4
71 95 8
30 20 7
3 44 4
17 88 5
64 2 7
49 44 7
68 88 2
60 94 6
81 30 7
86 12 10
10 59 8
62 52 5
37 1 5
25 81 7
5 37 1
27 95 1
11 37 1
14 76 3
58 86 5
31 88 9
35 31 9
66 1 1
65 30 5
93 10 7
91 87 9
26 55 4
29 2 10
54 81 5
78 25 9
46 62 4
84 55 7
63 20 10
9 59 4
53 97 4
98 65 3
61 97 2
21 60 8
70 60 1
82 76 6
83 59 9
56 91 5
41 61 6
7 55 5
74 71 5
69 55 6
57 25 8
99 1 7
77 3 8
32 61 3
28 91 5
96 2 2
19 12 8
75 13 6
42 65 6
18 3 5
67 5 2
80 19 3
33 21 8
23 71 2
100 94 3
90 9 8
73 62 3
89 23 10
4 80 2
48 78 6
43 49 8
51 89 7
50 10 1
6 17 2
40 75 7
85 33 5
34 32 1
45 10 8
22 52 10
39 90 8
36 78 8
8 44 5
79 10 9
16 76 7
24 30 2
47 63 10
15 87 7
