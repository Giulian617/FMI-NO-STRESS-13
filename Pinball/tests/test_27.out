857475067
383461998
163045986
739571499
788289778
982621729
771010761
617734128
599936712
951707017
893983267
784214821
836276935
959841251
513994302
189755095
373330774
776745058
12667560
505082916
505082916
985898995
117442323
692750188
39185495
567870214
819986743
467525688
707843578
725871096
742483865
647304309
959841251
593716586
741108382
842828970
61044239
788289778
82894260
36728448
888482527
285769048
891429416
849247959
949445676
144774714
511687078
400531124
132530354
373330774
222746488
355247085
617066420
193308221
357959356
